### ISSUE TITLE
Printing with powerpoint "notes" print layout does not show slide

### ISSUE THREAD

**Issue by benjanik** [gitbrent/PptxGenJS#446]
I generate presentations with a single image per slide, but after upgrading I can no longer print these presentations with the "notes" layout print option. You can see in my screenshot that nothing renders in the print preview when "notes" is selected as the layout option. This worked correctly in version 1.10.0, but I have recently upgraded to 2.4.0 in order to add slide notes. Example code included which worked for me in v1.10, I can supply images if need be (but any image should work).

![image](https://user-images.githubusercontent.com/830191/49482407-e7006600-f7e3-11e8-9d26-8da5f07d54fe.png)

```let pptx = new PptxGenJS();
this.slides.forEach(fileName => {
    let pptSlide = pptx.addNewSlide();
    pptSlide.addImage({
        x: 0,
        y: 0,
        w: 1.7777777778,
        h: 1,
        path: getFilePath(fileName),
        sizing: {
            type: 'cover',
            w: 10,
            h: 5.625,
        }
    });
});
return new Promise(resolve => {
    pptx.save('http', (buffer) => {
        resolve(buffer);
    });
});```

**Comment by benjanik** [comment]
Yes. It also prints if I manually change the notes master options. The slide image checkbox is off by default, and I didn't get the sense that the slide notes master was configurable from your library.

![image](https://user-images.githubusercontent.com/830191/49553553-162eda00-f8ad-11e8-94e3-126ac44fc18d.png)

**Comment by benjanik** [comment]
@gitbrent Any chance there you could add an option to control the notes master options in an upcoming release? Or perhaps you could point me in the right direction, and I could try to add this feature for you?

**Comment by gitbrent** [comment]
@benjanik I'm not even sure where to start with this one. 

You're absolutely welcome to dig through the unzipped PPTX file's Slides to see what the XML output should be versus what's being generated now and/or figure out which options are impacting this. 

Please submit a pull request when you get it working and i'll add it to the codebase.

**Comment by fpaupier** [comment]
Still interested in this, any idea where I could take a look ? Otherwise I'll dive in the xml
